lib/ask_web/controllers/channel_controller.ex

```diff
@@ -106,4 +106,20 @@ defmodule AskWeb.ChannelController do
 
     render(conn, "show.json", channel: channel |> Repo.preload([:projects, :user]))
   end
+
+  def pause(conn, %{"channel_id" => id}) do
+    channel_params = %{"paused" => true}
+    result = AskWeb.ChannelController.update(conn, %{"id" => id, "channel" => channel_params})
+    ChannelStatusServer.poll(ChannelStatusServer.server_ref())
+    ChannelStatusServer.wait(ChannelStatusServer.server_ref())
```

Comment:
The rationale for this is to force the status server to "learn the news" and stop the broker? I'm not sure we want to `wait` on that - do we need to block the users' interface for that?

It's probably enough if we just send the StatusServer a "recheck, please" message and let it handle it.